src/xdocs/config_javadoc.xml

```diff
@@ -3619,8 +3619,9 @@ public int foobar() {
           Checks that
           <a href="https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html#firstsentence">
           Javadoc summary sentence</a> does not contain phrases that are not recommended to use.
-          Summaries that contain only the <code>{@inheritDoc}</code> tag are skipped. Check also
-          violate Javadoc that does not contain first sentence.
+          Summaries that contain only the <code>{@inheritDoc}</code> tag are skipped. Summaries
+          that contain a non-empty {@code {@return}} are allowed. Check also violate Javadoc that
+          does not contain first sentence.
```

Comment:
make note that periods are not validated in return inline tags as it is set by javadoc tool.

Reply:
done

Reply:
@octylFractal , please let reviewer resolve review items as acknowledgement that it is done as requested.